install.sh

```diff
@@ -31,37 +31,39 @@ tmp_dir="$tmp/install.sh.$$"
 # do_download URL DIR
 do_download() {
   echo "Downloading $1"
-  if [[ $(type -t wget) ]]; then
+  if [[ $(command -v wget) ]]; then
     wget -q -O "$2" "$1" >/dev/null
-  elif [[ $(type -t curl) ]]; then
+  elif [[ $(command -v curl) ]]; then
     curl --fail -sSL -o "$2" "$1" &>/dev/null || true
   else
     error "Could not find wget or curl"
     return 1
   fi
 }
 
-# get_latest_version URL
+# get_latest_version
 get_latest_version() {
-  versionloc=${tmp_dir}/summon.version
-  versionfile=$(do_download "${1}" "${versionloc}")
-  if [ -f "${versionloc}" ]; then
-    local version=$(cat ${versionloc} | grep -o -e "[[:digit:]].[[:digit:]]*.[[:digit:]]*")
-    echo "${version}"
+  local latest_payload;
+  if [[ $(command -v wget) ]]; then
+    latest_payload=$(wget -q -O - "https://api.github.com/repos/cyberark/summon/releases/latest")
+  elif [[ $(command -v curl) ]]; then
+    latest_payload=$(curl --fail -sSL "https://api.github.com/repos/cyberark/summon/releases/latest")
+  else
+    error "Could not find wget or curl"
+    return 1
```

Comment:
Shouldn't this be `exit 1`? I feel like this will return a value that may be considered valid by the caller.